tests/cli/test_ci.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+
+# I guess that all of this could be moved to setup phase
+
+cat >> /etc/yum.repos.d/nightly.repo << __EOF__
+
+[RHEL-8-NIGHTLY-CRB]
+name=CRB
+baseurl=http://download-ipv4.eng.brq.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.2/compose/CRB/x86_64/os/
+enabled=1
+gpgcheck=0
+
+__EOF__
+
+yum-config-manager --enable epel epel-playground
+
+dnf -y install \
```

Comment:
This is exactly like the Dockerfile setup, it's probably time to move it to someplace common so that we are not keeping multiple file lists in sync. See PR #961

Reply:
Yes, that would be nice. I'll wait for a PR on rhel8-branch to be merged and rebase this PR then.

Reply:
#961 has been merged. rebase and use the common list of packages to be installed.

Reply:
I just realized that the list of packages is actually different for rhel-8 and for fedora-28 meaning we can't simply have one file list unless we either drop the testing on fedora-28 or make a conditional list or something...

Reply:
Well crud. And it looks like CentOS8 still doesn't have all the things we need either.
Which also made me realize that the CRB repo is internal only. We need to keep in mind that this code will eventually make its way to CentOS8 and it needs to be able to run in that environment as well.

The original plan was to use Fedora 28 to run things until CentOS8 was available and then to switch over to it.

If we are going to use RHEL repos they need to be passed to the tests somehow, not hard-coded into them.